content/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Hello World</title>
+  <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
+  <script>
+  $("document").ready(function(){
+    $(".outerbox").css({"height": ($(window).height() - 20) + "px"});
+  });
+  </script>
+  <link rel="icon" type="image/x-icon" href="https://giantswarm.io/static/img/favicon32.ico">
```

Comment:
The SVG is included. The icons not. I would go for one route. Probably include all files.